build-logic/gobley-gradle-cargo/src/main/kotlin/CargoPlugin.kt

```diff
@@ -280,7 +304,7 @@ class CargoPlugin : Plugin<Project> {
     }
 
     private fun Project.configureJvmPostBuildTasks(
```

Comment:
Is this now just `configurePostBuildTasks`?

Reply:
No, it's still `configureJvmPostBuildTasks`, but `AbstractKotlinTarget` is the closest common ancestor of `KotlinJvmTarget` and `KotlinWithJavaTarget`. `AbstractKotlinTarget` has no Java-specific methods, so I just changed the parameter type to `KotlinTarget`, which is the super-interface of all Kotlin targets.